lib/config/default.js

```diff
@@ -313,16 +357,35 @@ exports.config = () => ({
      * Relic servers. The memory concerns are something you should consider for
      * your own server's sake. The payload of events is compressed, but if it
      * grows too large the New Relic servers may reject it.
-     *
-     * @env NEW_RELIC_ERROR_COLLECTOR_MAX_EVENT_SAMPLES_STORED
      */
-    max_event_samples_stored: 100,
+    max_event_samples_stored: {
```

Comment:
Should this one get the 
`env: 'NEW_RELIC_ERROR_COLLECTOR_MAX_EVENT_SAMPLES_STORED' ` 
property?

Reply:
the env property is only designed to override a default convention.  The env var follows the convention which is why there isn't an override specified.  `NEW_RELIC_ERROR_COLLECTOR_MAX_EVENT_SAMPLES_STORED` `config.error_collector.max_event_samples_stored`

Reply:
this part of the config stanza does have a bunch of overrides, so I can see why you ask the question